index.js

```diff
@@ -4,7 +4,31 @@
  * Depending on the operation, either add up all of the numbers or subtract all of the numbers, from left to right.
  * @returns {number} The result of either adding all numbers or subtracting all numbers, depending on the arguments added to the command line.
  */
-function calculator() {}
+function calculator() {
+    let array = process.argv.slice(2) //just to simplify the array being worked on
+    let result = Number(array[1]); //anchor the first number for the minus and divide
+    if (/\d/g.test(array[0]) || !array[0]){//found a function to check strings for a number with regex
+        return 'No operation provided...'
+    }else if (!array[1]){ //checks if first item after operator exists
+        return 'No numbers provided...'
+    }else if (array[0] === 'plus') { //check first word for the operation keywords
+        for (let i =2; i< array.length; i++){ //i starts at 2 since result contains i=1
+            result += Number(array[i])
+        }
+    }else if (array[0] === 'minus') {
+        for (let i =2; i< array.length; i++){
+            result -= Number(array[i])
+        }
+    }else if (array[0] === 'divide') {
```

Comment:
The instructions only specify plus and minus. If your user uses divide it should provide an 'invalid operator' message.